documentation/NewDataRequirements.md

```diff
@@ -18,11 +18,21 @@ The metadata file **must** follow these rules:
   * **max** (Integer)
   * **enumerations** (List of codes)
 * The `dataset` CDE is required.
+* In the parent dictionary there will be a `version` property, of string format.
 
```

Comment:
You should add the longitudinal flag here too.

Reply:
I was thinking that by default it's not needed, so we won't need to change all the previous metadata files.

If it's not there we will assume that `longitudinal` = false.